<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurrencyConversionView</class>
 <widget class="QWidget" name="CurrencyConversionView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>300</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>148</width>
     <height>204</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdit2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Arial Unicode MS</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Arial Unicode MS&quot;;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox2">
          <item>
           <property name="text">
            <string>人民币</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>美元</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>欧元</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>英镑</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>澳元</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>港币</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Arial Unicode MS&quot;;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox1">
          <item>
           <property name="text">
            <string>人民币</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>美元</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>欧元</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>英镑</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>澳元</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>港币</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>17</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
